scriptworker/test/test_task.py

```diff
@@ -150,6 +150,23 @@ def test_reclaim_task_non_409(context, successful_queue, event_loop):
 
 
 @pytest.mark.asyncio
+async def test_run_task_negative_11(context, mocker):
```

Comment:
We lost the `@pytest.mark.asyncio` for the next test below with this change. I pushed a fix for this, just moved this up. Thank you!

Reply:
oops. Sorry for the mix up. You're welcome :)